doc/AWSSQSConnector.md

```diff
@@ -0,0 +1,232 @@
+   */
+  MD5OfMessageAttributes?: string
+  /**
+   * Each message attribute consists of a Name, Type, and Value. For more information, see Amazon SQS Message Attributes in the Amazon Simple Queue Service Developer Guide.
+   */
+  MessageAttributes?: MessageBodyAttributeMap
+}
+```
+
+_Example:_
+
+```js
+(event, app) => { 
+  console.log('new message received', event)
+  // 'new message received', { MessageId: '71c06956-76e1-4a8c-bc00-ed910566f36e', ReceiptHandle: 'AQEBTL1CTtn1clJ0XMSmRtpz7...', MD5OfBody: '9a72c70562843b823c2c9cad30665fe4', Body: 'Message from Reshuffle to queue' }
+}
+```
+
+This event is triggered when new messages are received from the queue.
+Reshuffle checks for new messages every minute.
+Per default, it deletes the messages from the queue once retrieved. 
+Set `deleteAfterReceive` to `false` to prevent this.
+
+__Note:__ if you do so, events can be triggered several times for the same message depending on your queue strategy.
+
+#### Connector actions
+
+Most of the actions are provided via the [sdk](#sdk).
+
+##### <a name="sendMessage"></a>Send message
+
+_Definition:_
+
+```ts
+(params: AWS.SQS.Types.SendMessageRequest) => Promise<PromiseResult<AWS.SQS.SendMessageResponse, AWS.AWSError>>
+
+export interface SendMessageRequest {
+  /**
+   * The URL of the Amazon SQS queue to which a message is sent. Queue URLs and names are case-sensitive.
+   */
+  QueueUrl: string
+  /**
+   * The message to send. The minimum size is one character. The maximum size is 256 KB.  A message can include only XML, JSON, and unformatted text. The following Unicode characters are allowed:  #x9 | #xA | #xD | #x20 to #xD7FF | #xE000 to #xFFFD | #x10000 to #x10FFFF  Any characters not included in this list will be rejected. For more information, see the W3C specification for characters.
+   */
+  MessageBody: string
+  /**
+   *  The length of time, in seconds, for which to delay a specific message. Valid values: 0 to 900. Maximum: 15 minutes. Messages with a positive DelaySeconds value become available for processing after the delay period is finished. If you don't specify a value, the default value for the queue applies.   When you set FifoQueue, you can't set DelaySeconds per message. You can set this parameter only on a queue level.
+   */
+  DelaySeconds?: Integer
+  /**
+   * Each message attribute consists of a Name, Type, and Value. For more information, see Amazon SQS Message Attributes in the Amazon Simple Queue Service Developer Guide.
+   */
+  MessageAttributes?: MessageBodyAttributeMap
+  /**
+   * The message system attribute to send. Each message system attribute consists of a Name, Type, and Value.    Currently, the only supported message system attribute is AWSTraceHeader. Its type must be String and its value must be a correctly formatted AWS X-Ray trace header string.   The size of a message system attribute doesn't count towards the total size of a message.
```

Comment:
`header string.   The s` - double space?

Reply:
that's copied from AWS doc, but I've changed it